src/openapi-generator.ts

```diff
@@ -60,21 +60,27 @@ const openApiVersions = ['3.0.0', '3.0.1', '3.0.2', '3.0.3', '3.1.0'] as const;
 
 export type OpenApiVersion = typeof openApiVersions[number];
 
-export type OpenAPIObjectConfig = Omit<
-  OpenAPIObject,
-  'paths' | 'components' | 'webhooks' | 'openapi'
->;
-
 interface ParameterData {
   in?: ParameterLocation;
   name?: string;
 }
 
+// TODO: Can I improve upon the anys
```

Comment:
Well, at least the return types are actually "any" - can be anything. Or map them to `unknown`